docs/topics/modules/reviewing_your_staged_application.adoc

```diff
@@ -0,0 +1,26 @@
+[id="reviewing_your_staged_application"]
+= Reviewing your staged application
+
+When you create a new codebase, the selected pipeline builds your application. It pushes version 1.0.1 of your new application into the _Stage_ environment, and then awaits your approval to deploy into the _Run_ environment.
+
+<<about_pipelines_stage_run,_Stage_ and _Run_>> are individual OpenShift projects. _Stage_ is a production staging area to review and test changes before they are finalized and then staged on _Run_.
+
+To review your application on _Stage_:
+
+. At the top of the page, click *Create*, and then click *Pipelines* to see the build pipelines for your new application. The pipeline first sets your build server, starts the build, and releases the build. When the pipeline build pushes the application to stage, it displays at the *Approve* stage. This could take a few minutes.
```

Comment:
The second sentence here again suggests that the pipeline is the actor. I don't think the pipeline itself is doing anything, AFAIK. 
Also, could should not be mentioned in docs as well. Anything that indicates uncertainty should not be there, such as "may", "should", "could", "would", etc.

Reply:
How about this?
When you create a new codebase, the selected pipeline is triggered. In the **Build Release** stage of the pipeline, the build server is set, the build is started and then released.
+1, weeded out the "may" and "could", mercifully did not find any should or would :)